app/policies/story_policy.rb

```diff
@@ -0,0 +1,2 @@
+class StoryPolicy < ApplicationPolicy
+end
```

Comment:
I had to create all the policies to be able to use this Pundit feature in the admin panel